### DISCUSSION TITLE
OPTIONS TEMP behaving strange

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#1025] [Q&A]
Again, I have an issue with .OPTIONS TEMP when it is not 27, but e.g. 26 or 28 or anything else.
The schematic is a simple audio power amp, graphs are from the output voltage (out) and the current thru the output transistors (pr).
Below are graphs looking like they should, at TEMP=27.
![MFA1v9-debug_pr_27C](https://github.com/user-attachments/assets/c23d5212-0f21-4ac6-8959-fd4e68565aad)
![MFA1v9-debug_out_27C](https://github.com/user-attachments/assets/1859305f-e0cb-4eb8-a668-e5a0f9463244)
Below are the same graphs when TEMP = 28.
![MFA1v9-debug_pr_28C](https://github.com/user-attachments/assets/2540d60a-1836-42ef-b9a2-b409dfadc313)
![MFA1v9-debug_out_28C](https://github.com/user-attachments/assets/9195a94d-f10b-426c-bd6e-d0d4a5b9e04c)
The schematic is not complicated, all devices are from BJT_Extended.
[MFA1v8debug.zip](https://github.com/user-attachments/files/17528017/MFA1v8debug.zip)
It is clearly not physical to have this change in behaviour for 1 degree temperature change.
If I start again and rebuild the schematic from scratch it is ok for a while, but the anomaly returns after working with TRANS, DISTOR and AC sims for a while.
I would be happy if someone could look into this.
I have not been able to correlate this with any specific action from my side.

**Comment by tomhajjar** [comment]
Your sb5100 has a problem...

[2024-10-25_215129](https://github.com/user-attachments/assets/580c7385-55b2-4dc4-b88c-e8e8740a9e67)

**Comment by bnilsson11** [comment]
My bad, please delete the sb5100, I forgot to remove them.It is not the cause of the problem.The TEMP anomaly is there without it.If I claim that all devices are from BJT_Extended, it should be true.Sorry.MvhBengt NilssonSkickat från min iPhone26 okt. 2024 kl. 04:35 skrev tomhajjar ***@***.***>:﻿
Your sb5100 has a problem...
2024-10-25_215129
